modules/specialized-guides/pages/salt/salt-formula-liberate.adoc

```diff
@@ -278,7 +278,7 @@ This method makes use of the bootstrap script created above for onboarding new s
 
 . In the {productname} {webui}, open the System Details page of any registered client you want to migrate to {sll}.
 
-. Click the [guimenu]`Reactivation`` tab.
+. Click the [guimenu]``Reactivation`` tab.
```

Comment:
```suggestion
. Click the [guimenu]`Reactivation` tab.
```

Reply:
I think we still want to stick with double backtics here.